java/ql/src/semmle/code/java/dataflow/internal/TaintTrackingUtil.qll

```diff
@@ -680,3 +675,18 @@ private MethodAccess callReturningSameType(Expr ref) {
   ref = result.getQualifier() and
   result.getMethod().getReturnType() = ref.getType()
 }
+
+private import semmle.code.java.dataflow.FlowSources
```

Comment:
We tend to keep file-module imports at the top of the file. So this line can just be moved to the other imports at the top.

Reply:
I think I (wrongly) followed the style of other imports that aren't at the top, but they are probably not file-module imports. Fixed in 2c769fe, thanks!